sample-plugin/pom.xml

```
@@ -454,6 +454,13 @@
         </plugins>
     </build>
     <profiles>
+        <profile>
+            <id>2.332.x</id>
+            <properties>
+                <bom>2.332.x</bom>
+                <jenkins.version>2.332.1</jenkins.version>
```

this should be the weekly version till the LTS is released
```suggestion
                <jenkins.version>2.332</jenkins.version>
```